packages/arb-token-bridge-ui/src/token-bridge-sdk/hooks/useArbTokenBridge.ts

```diff
@@ -35,13 +40,21 @@ import {
   NodeBlockDeadlineStatusTypes
 } from './arbTokenBridge.types'
 import { useBalance } from './useBalance'
-import { getUniqueIdOrHashFromEvent } from '../util/migration'
-import { getL1TokenData, isClassicL2ToL1TransactionEvent } from '../util'
+import { getL1TokenData, getL1ERC20Address } from '../../util/TokenUtils'
 
 export const wait = (ms = 0) => {
   return new Promise(res => setTimeout(res, ms))
 }
 
+function isClassicL2ToL1TransactionEvent(
+  event: L2ToL1TransactionEvent
+): event is EventArgs<ClassicL2ToL1TransactionEvent> {
+  return (
+    typeof (event as EventArgs<ClassicL2ToL1TransactionEvent>).batchNumber !==
```

Comment:
Not a fan of the casting here. If we cast to the type, `batchNumber` is present in the type so it can't be `undefined`, giving as a false sense of security. I would revert to the previous implementation.

Reply:
Reverted